### PR TITLE
Cleanup updates

### PR DESCRIPTION
**Pull request by jweakley**
- Simplified autogenerated files. 
- Corrected install bugs.
- Pinned library version for compatibility.

### CODE REVIEW

**Comment by jweakley** [comment]
- Fixes #32
- Fixes #31 
- Fixes #17 
- Fixes #6 
- Fixes #7

**Comment by chadzink** [comment]
Looks ok, I am not familiar with the application or the configuration. May want Sam to take a look at this.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (Samsinite) [inline .gitignore]: Lets add `*.swp` to this as well.

File: boilerplates/ember-cli-index.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<%= render_ember_app :frontend do |head, body| %>
+  <% head.append do %>
+    <%= csrf_meta_tags %>
+  <% end %>
+  <% body.append do %>
+    <% if Rails.application.secrets.wildland_server_status == 'development' %>
+      <div id="pipeline-flag">
+        <div class="development-flag">
+          <p>Remio Development</p>
```
Comment (Samsinite) [inline boilerplates/ember-cli-index.erb]: Remove `Remio` from here...

File: boilerplates/ember-cli-index.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        </div>
+      </div>
+    <% end %>
+    <% if Rails.application.secrets.wildland_server_status == 'staging' %>
+      <div id="pipeline-flag">
+        <div class="staging-flag">
+          <p>Remio Staging</p>
```
Comment (Samsinite) [inline boilerplates/ember-cli-index.erb]: Ditto

File: template.rb [hunk 4]
```diff
@@ -8,31 +8,31 @@
 # This template assumes you have followed the setup guide at:
 # https://github.com/wildland/guides/#setting-up-your-development-enviroment
 #
-ruby_version = '2.2.3'
-node_version = 'v5.4.0'
+ruby_version = '2.3.1'
+node_version = 'v4.5.0'
```
Comment (Samsinite) [inline template.rb]: Why are we rolling back node here?

**Comment by Samsinite** [comment]
:+1: after addressing comments

File: template.rb (hunk 4 above)
Comment (jweakley) [inline template.rb]: https://github.com/nodejs/LTS#lts-schedule and https://devcenter.heroku.com/articles/nodejs-support#supported-runtimes.  
TL;DR 5 isn't included in LTS. 
We could bump forward to 6.x.x if you think that makes more sense.

Comment (Samsinite) [inline template.rb]: Nah, no problem with stick the default at v4.5 for now

Comment (jweakley) [inline template.rb]: Ok. I'll go ahead an push this then.
